src/secondaryServer/secondaryserver.py

```diff
@@ -111,14 +115,14 @@ def writeToServer(self, action, *args, **kw):
         out.writeUInt32(0)
         out.writeQString(action)
         
-        for arg in args :            
-            if type(arg) is IntType:
+        for arg in args:
+            if type(arg) is type(int):
```

Comment:
found no case with args during debug .. so not really tested.
but your right, it's wrong .. seems right at the time .. hmm.
how can it be fixed.. another fix-commit?